### ISSUE TITLE
Hello, it's not that semi supervised learning like mixmatch can't be used on unbalanced data sets？

### ISSUE THREAD

**Issue by ghost** [iBelieveCJM/Tricks-of-Semi-supervisedDeepLeanring-Pytorch#5]
Hello, it's not that semi supervised learning like miamache can't be used on unbalanced data sets. I use it on my own data set.  the test set accurate rises by 2 epochs and then decreases continuously. However, the loss of training set is declining. What's the matter? Is my dataset too unbalanced
